bin/tls-include.sh

```diff
@@ -105,15 +105,22 @@ function create_tls_certs_cm {
   for app in "${apps[@]}"; do
     # Only create the secrets if they do not exist
     TLS_SECRET_NAME=$app-tls-secret
-    if [ -z "$(kubectl get secret -n $namespace $TLS_SECRET_NAME -o name 2>/dev/null)" ]; then
-        if [ "$deployedIssuers" == "false" ]; then
-          deploy_issuers $namespace $context
-          deployedIssuers="true"
-        fi
-        deploy_app_cert "$namespace" "$context" "$app"
-    else
-      log_debug "Using existing $TLS_SECRET_NAME for [$app]"
+    if [ -n "$(kubectl get secret -n $namespace $TLS_SECRET_NAME -o name 2>/dev/null)" ]; then
+       if kubectl get secret -n $namespace $TLS_SECRET_NAME -o "jsonpath={.data['tls\.crt']}" | base64 -d | openssl x509 -enddate -noout -checkend 86400
```

Comment:
This will pull the secret down, decode the cert, and then use openssl command to check if it is either expired or will expire within the next 24 hours.

1. Any issues with checking within 24 hours, should we configure a different time frame?
2. `openssl` can be used to check the cert here even if it was issued by certmanager. don't think thats an issue to use openssl even though this is the `cert-manager` flow but just wanted to point that out